bench/versus/Micro/PkgStore.hs

```diff
@@ -0,0 +1,15 @@
+{-# OPTIONS_GHC -fno-warn-orphans -fsimpl-tick-factor=500 #-}
+module Micro.PkgBinary (serialise, deserialise) where
+
+import Micro.Types
+import Data.Store as Store
+import Data.ByteString.Lazy as BS
+
+serialise :: Tree -> BS.ByteString
+serialise pkgs = BS.fromStrict (Store.encode pkgs)
+
+deserialise :: BS.ByteString -> Tree
+deserialise = (\(Right x) -> x) . Store.decode . BS.toStrict
```

Comment:
Mm, we should think about how to do fair comparison between libs when one of them forces the use of strict bytestrings.